src/components/Book.js

```diff
@@ -0,0 +1,34 @@
+/* eslint-disable react/prop-types */
+import { useDispatch } from 'react-redux';
+import { removeBook } from '../redux/books/books';
+
+const Book = (props) => {
+  const dispatch = useDispatch();
+  const { title, author, id } = props;
+
+  return (
+    <div className="books-container">
+      <div className="book-info">
+        <h4 className="book-category">Economy</h4>
+        <h2 className="book-name">{title}</h2>
+        <p className="book-author">{author}</p>
+        <div className="book-buttons-container">
+          <button type="button" className="book-button">Comments</button>
+          <button type="button" className="book-button" onClick={() => dispatch(removeBook(id))}>Remove</button>
```

Comment:
- [ ] Same here as well it's better to create the function that handles removing the book outside of the return then call it here, kindly adjust that.

Reply:
Fixed.